server/src/test/java/org/candlepin/controller/ActivationKeyControllerTest.java

```diff
@@ -47,29 +46,27 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.Mockito.eq;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 /**
- * ActivationKeyResourceTest
+ * ActivationKeyControllerTest
  */
-public class ActivationKeyResourceTest extends DatabaseTestFixture {
-    @Inject private ProductCurator productCurator;
+public class ActivationKeyControllerTest extends DatabaseTestFixture {
```

Comment:
As you have mentioned in comments above, we need proper Resource unit tests because our current *ResourceTests actually test the logic (which now resides in the controller). Having moved ActivationKeyResourceTest to ActivationKeyControllerTest, it is probably a good idea to create a proper ActivationKeyResourceTest.java now, that tests the resource stuff (for example, test the new validation `@NotEmpty, `@NotNull`, `@Positive` validation added).